ansible_events/drools/durable_rules_engine.py

```diff
@@ -0,0 +1,230 @@
+import json
+import logging
+
+import requests
+
+_logger = logging.getLogger(__name__)
+
+
+class DurableRulesEngine:
+    __host = None
+    __last_resp = None
```

Comment:
Please initialize instance attributes in the `__init__` method instead of creating class attributes with a default value.